### DISCUSSION TITLE
How can I focus on a window?

### DISCUSSION THREAD

**Discussion by khaitranhq** [kovidgoyal/kitty#7232] [Q&A] [answered]
For example, I launched some windows in a tab. How can I focus on a specific window? In other words, I want to have some keybinding to make a window fullscreen to focus on coding and restore it to the previous position after that.

**Comment by kovidgoyal** [comment, marked as answer]
See https://sw.kovidgoyal.net/kitty/actions/#action-toggle_layout

**Comment by khaitranhq** [comment]
With my config, `toggle_layout stack` doesn't work with splits layout
```
enabled_layouts splits
map alt+shift+f toggle_layout stack

```

**Reply by khaitranhq** [reply]
I changed the `enabled_layouts` to `splits,stack` and it worked. Thank you @kovidgoyal
